meson_options.txt

```diff
@@ -26,6 +26,12 @@ option('rootprefix', type : 'string',
        description : '''override the root prefix''')
 option('link-udev-shared', type : 'boolean',
        description : 'link systemd-udev and its helpers to libsystemd-shared.so')
+option('static-libsystemd', type : 'combo',
+       choices : ['true', 'false', 'pic', 'no-pic'], value : 'false',
```

Comment:
We generally just put the default first in the choices list, and do not specify it explicitly (mostly to make this all a bit shorter).

Reply:
nitpick: Having "true" and "false" together with other options sounds a lot like [using a boolean as a tri-state](https://thedailywtf.com/articles/What_Is_Truth_0x3f_)

Can we have a name this option differently (not really sure what to suggest) so that the choices could be "shared", "static", "pic" and "no-pic" instead? I think that would be slightly cleaner...

Reply:
It's not a choice between "shared" and "static", this option only governs the static library. We *could* split this in two options: -Dstatic-libsystemd=true|false, -Dstatic-libsystemd-pic=true|false, to avoid, but I don't think it would be any better. I think it's totally OK to have multiple "true" values.